@xen-orchestra/lite/src/components/Console.tsx

```diff
@@ -49,11 +63,13 @@ const Console = withState<State, Props, Effects, Computed, ParentState, ParentEf
         url.protocol = window.location.protocol === 'https:' ? 'wss:' : 'ws:'
         url.searchParams.set('session_id', xapi.sessionId)
 
-        // eslint-disable-next-line no-new
-        new RFB(this.state.container.current, url, {
+        this.state.RFB = new RFB(this.state.container.current, url, {
           wsProtocols: ['binary'],
         })
       },
+      sendCtrlAltDel: function () {
+        confirm('Send Ctrl+Alt+Del to VM?') && this.state.RFB?.sendCtrlAltDel()
```

Comment:
Please create a modal component for this (even if it's a very simple one for now) and internationalize the message.